components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/listeners/UserManagementAuditLogger.java

```diff
@@ -96,11 +100,16 @@ public boolean doPreSetUserClaimValue(String userName, String claimURI, String c
 
         if (isEnable()) {
             JSONObject dataObject = new JSONObject();
+            if (LoggerUtils.isLogMaskingEnable) {
+                String sanitizedClaimValue = LoggerUtils.getMaskedClaimValue(claimURI, claimValue);
```

Comment:
```suggestion
                String maskedClaimValue = LoggerUtils.getMaskedClaimValue(claimURI, claimValue);
```

Please check all applicable places